tests/test_basic_splitter.py

```diff
@@ -0,0 +1,83 @@
+from draive import split_text
+
+
+def test_text_small_enough() -> None:
+    sample_text: str = "abcd efg hij"
```

Comment:
I gather it's a _Kacper's_ thing but there's no need to annotate a variable when you assign a value of known type.

Reply:
there is no need but it is a good practice 😉